config/v1/types_authentication.go

```diff
@@ -291,6 +310,26 @@ type TokenIssuer struct {
 	//
 	// +optional
 	CertificateAuthority ConfigMapNameReference `json:"issuerCertificateAuthority"`
+	// discoveryURL is an optional field that, if specified, overrides the default discovery endpoint
+	// used to retrieve OIDC configuration metadata. By default, the discovery URL is derived from `issuerURL`
+	// as "{url}/.well-known/openid-configuration".
```

Comment:
If we are referencing other fields, lets use the proper field name
```suggestion
	// as "{issuerURL}/.well-known/openid-configuration".
```